libs/shared/ui-components/src/Team/Team.tsx

```diff
@@ -21,8 +21,8 @@ export const Team: FC<TTeamProps> = ({
     <section className="px-[1.8rem] my-[6rem] mx-auto md:my-[8rem] lg:px-[3rem] xl:px-[18rem] max-w-layout">
       <h2
```

Comment:
I decided to revert this back to an h2 instead of an h1 because it should be an h2 on consulting/about-us. It could be an h1 on labs/team because there is no h1 current on that page. However, since there are several pages that do not have an h1 (e.g., Consulting home page), I decided that for now it's better to have a page without an h1 than to have false hierarchy on the Consulting about us page.